File: Data Structures & Algorithms/unique-email-addresses/submission-7.py
class Solution:
    def numUniqueEmails(self, emails: List[str]) -> int:
        uniqueEmails = set()

        for email in emails :
            atPos = email.find('@')
            local = email[0:atPos]
            domain = email[atPos:]

            cleanLocal = ""

            for c in local :
                if c == '+' :
                    break
                if c != '.' :
                    cleanLocal += c

            finalEmail = cleanLocal + domain;
            uniqueEmails.add(finalEmail);

        return len(uniqueEmails);

File: Data Structures & Algorithms/arranging-coins/submission-7.cpp
class Solution {
public:
    int arrangeCoins(int n) {

        if(n == 1) return 1;
        for(long long i = 1 ; i <= n ; i++)
        {
            if( (i)*(i+1)/2 >= n)
                return i-1;
        }

        return -1;
    }
};